listenbrainz/mbid_mapping/docker/crontab

```diff
@@ -3,3 +3,10 @@
 
 # Run the huesound color sync hourly
 10 * * * * listenbrainz /usr/local/bin/python /code/mapper/manage.py update-coverart >> /code/mapper/lb-cron.log 2>&1
+
+
+# Build the mb metadata cache from scratch on first thursday of every month, MB dumps run on Wed and Sat so avoid those days
+0 15 * * 4 listenbrainz [ $(date +\%d) -le 7 ] && /usr/local/bin/python /code/mapper/manage.py cron-build-mb-metadata-cache >> /code/mapper/lb-cron.log 2>&1
+
+# Update the mb metadata cache incrementally every 4 hours
+0 */4 * * * listenbrainz /usr/local/bin/python /code/mapper/manage.py update-mb-metadata-cache >> /code/mapper/lb-cron.log 2>&1
```

Comment:
should we have a way to stop this from running when the main cache is updating? I guess the swap to the new tables happens in a single transaction, but does it make sense to perform updates when the table is about to be deleted?

Reply:
I was thinking about that but figured it wouldn't be straightforward to do so. We'll have to add locks etc like we have in LB main cron. So inclined to going ahead with this for now. If some issue comes up, reevaluate.